src/HUDTeam/DrawManagerImpl.java

```diff
@@ -20,11 +20,11 @@ public static void drawLevel(final Screen screen, final int level){
         backBufferGraphics.drawString(levelText, xPosition - fontRegularMetrics.stringWidth(levelText) / 2, yPosition); // edit by jesung ko - TeamHUD
     } // Lee Hyun Woo - level
     
-    public static void drawBulletSpeed(final Screen screen, final int bulletSpeed) {
+   public static void drawAttackSpeed(final Screen screen, final double attackSpeed) {
         backBufferGraphics.setFont(fontRegular);
         backBufferGraphics.setColor(Color.WHITE);
-        String bulletSpeedText = String.format("BS : %d px/f ", bulletSpeed);
-        backBufferGraphics.drawString(bulletSpeedText, 10, screen.getHeight() - 15);
+        String attackSpeedText = String.format("AS : %.2f ", attackSpeed);
```

Comment:
its old code
Invalid conflict resolution.

Reply:
I checked it now.
This will be solved on new pr.